packages/select/src/select.vue

```diff
@@ -365,6 +365,12 @@
         }
       },
 
+      query(val, oldVal) {
+        if (this.filterable && val !== oldVal) {
```

Comment:
val & oldVal comparison is unnecessary, vue already guarantees that when you use watch.

Reply:
Indeed, thank you. Removed.